src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.TrackingSession.cs

```diff
@@ -179,6 +179,13 @@ public void Cancel()
                         }
                         else
                         {
+                            // We do not yet support renaming (inline rename or rename tracking) on
```

Comment:
I'll include the fact that these should be unified in my write-up. Thanks for pointing it out.